test/coverlet.core.tests/Reporters/CoberturaReporterTests.cs

```diff
@@ -146,43 +146,65 @@ public void TestReportWithTwoDifferentDirectories()
 
             string absolutePath1;
             string absolutePath2;
+            string absolutePath3;
+            string absolutePath4;
+            string absolutePath5;
+            string absolutePath6;
 
             if (isWindows)
             {
-                absolutePath1 = @"C:\projA\file.cs";
-                absolutePath2 = @"E:\projB\file.cs";
+                absolutePath1 = @"C:\projA\dir1\dir10\file1.cs";
```

Comment:
add test for `F:\file1.cs`

Reply:
and maybe also empty roots `F:\` `/` if there is an error on modules data...error should be clear no out of range or cryptic one.

Reply:
Done. Implementation can handle empty roots without throwing.